download-clis.sh

```diff
@@ -34,4 +34,13 @@ if [ "$ARCH" = "amd64" ]; then
   [[ ! -f "downloads/helm-linux-amd64" ]] && echo "download helm failed" && exit -1
   echo "Downloaded helm v3 to downloads/"
 
+  echo "Downloading subctl v0.9.0"
+  assetURL=$(curl -Ls -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/submariner-io/submariner-operator/releases | jq -r '.[] | select(.tag_name=="v0.9.0") | .assets[] | select(.name|contains("linux-amd64")) | .browser_download_url')
+  # echo " assetURL is $assetURL"
+  curl -fksSL $assetURL | tar -xvJ -C ./downloads/ subctl-release-0.9-198fe39/subctl-release-0.9-198fe39-linux-amd64
```

Comment:
Do we need to worry about vendorizing these dependencies?

Reply:
No, the upstream build pulls newer versions each time for `oc`, `kubectl` and `helm`.  For `subctl` they do not have it available yet as a `latest` tag, so when we want to pull a newer version, this shell script will need to be updated.  I have asked the team to think about making the naming scheme more predictable than including the commit SHA as part of the name.